ldms/src/ldmsd/ldmsd_request.c

```diff
@@ -774,16 +797,20 @@ int ldmsd_handle_request(ldmsd_req_ctxt_t reqc)
 	uid_t luid;
 	gid_t lgid;
 	mode_t mask;
+	int rc;
 
 	__dlog("handling req %s\n", ldmsd_req_id2str(reqc->req_id));
 
 	/* Check for request id outside of range */
 	if ((int)request->req_id < 0 ||
-	    request->req_id >= (sizeof(request_handler)/sizeof(request_handler[0])))
+		request->req_id >= (sizeof(request_handler)/sizeof(request_handler[0])))
```

Comment:
I'm not a fan of this indenting change.

Reply:
I think this an accidental change, please remove.